### PR TITLE
[TS] Added outputGlobalTypes option to specify path for global types

### PR DESCRIPTION
**Pull request by danilobuerger**

### CODE REVIEW

**Comment by williamboman** [comment]
This is great! I think it would also be nice to be able to set the filename (`globalTypes.ts`).

**Comment by danilobuerger** [comment]
@williamboman Thats exactly what this option is for, the path includes the filename

**Comment by williamboman** [comment]
Oh shit, my bad!

**Comment by zionts** [comment]
@danilobuerger thanks for submitting this! Could you update https://github.com/apollographql/apollo-cli/blob/master/CHANGELOG.md with what this PR is doing?

File: packages/apollo-cli/src/commands/codegen/generate.ts [hunk 1]
```diff
@@ -95,6 +95,10 @@ export default class Generate extends Command {
       description:
         'By default, TypeScript/Flow will put each generated file in a directory next to its source file using the value of the "output" as the directory name. Set "outputFlat" to put all generated files in the directory relative to the current working directory defined by "output".'
     }),
+    outputGlobalTypes: flags.string({
```
Comment (zionts) [inline packages/apollo-cli/src/commands/codegen/generate.ts]: Not doing a full review, but I think that `outputGlobalTypes` is a strange name (and reads like a boolean) for a string that is meant to specify a file location to write global type information to. Perhaps something like `globalTypesFilePath`?

**Comment by danilobuerger** [comment]
@zionts In this repo the Changelog is done on Publish. See https://github.com/apollographql/apollo-cli/commits/master/CHANGELOG.md not during PRs.

File: packages/apollo-cli/src/commands/codegen/generate.ts (hunk 1 above)
Comment (danilobuerger) [reply packages/apollo-cli/src/commands/codegen/generate.ts]: The intention was that it is aligned with the other output options (`output` and `outputFlat`) for easier grouping if they all begin with `output...`. But I really don't care what the param is named :-)

Comment (martijnwalraven) [reply packages/apollo-cli/src/commands/codegen/generate.ts]: I agree `outputGlobalTypes` feels a bit weird as a flag name. Maybe `globalTypesFile`?

Comment (danilobuerger) [reply packages/apollo-cli/src/commands/codegen/generate.ts]: @martijnwalraven I amended the commit

**Comment by rahuljaindream11** [comment]
Will this key work for swift target

**Pull request merged**